styles.css

```diff
@@ -0,0 +1,92 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  width: 375px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  width: 375px;
+}
+
+#mylogo {
+  width: 76px;
+  height: 20px;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  text-decoration: none;
+  margin-left: 40px;
+}
+
+#humberger-bottun {
+  width: 20px;
+  cursor: pointer;
+  height: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  margin-right: 45px;
+}
+
+.ico {
+  width: 100%;
+  height: 3px;
+  background-color: #6070ff;
+}
+
+main {
+  width: 375px;
+  height: 550px;
+  background-image: url("/image/header-shapes\ mobile@2x.svg");
```

Comment:
- [ OPTIONAL] You have done well adding the images :clap: , kindly make sure they also display on remote platfom as raw.githack to make sure they will be displayed in the future when you deploy __as for now we can only see the icons and images when we clone and locally__